roles/addons/users_basic/molecule/default/converge.yml

```diff
@@ -0,0 +1,24 @@
+---
+- name: Converge
+  hosts: all
+
+  vars:
```

Comment:
Open discussion: is there a recommendation to define variables to test the role in converge.yml?

I read the [Molecule documentation](https://molecule.readthedocs.io/en/latest/configuration.html#id12 ) and noticed we can define an inventory for the scenario in the Provisioner:

```yaml
provisioner:
  name: ansible
  inventory:
    group_vars:
      foo1:
        foo: bar
      foo2:
        foo: bar
        baz:
          qux: zzyzx
    host_vars:
      foo1-01:
        foo: bar
```
This is the implementation I followed for #171 ([molecule.yml](https://github.com/bluebanquise/bluebanquise/blob/49b37ea954e254c2220bf9107b87d8d677e22bab/roles/core/repositories_client/molecule/default/molecule.yml)).

We can discuss this in Slack.

Reply:
as discussed in slack, I would like to postpone this conversation until we get a proper default inventory for localhost molecule tests.